x-pack/packages/kbn-elastic-assistant/impl/assistant/assistant_overlay/index.tsx

```diff
@@ -63,15 +64,15 @@ export const AssistantOverlay = React.memo(() => {
   const handleShortcutPress = useCallback(() => {
     // Try to restore the last conversation on shortcut pressed
     if (!isModalVisible) {
-      setConversationId(localStorageLastConversationId ?? WELCOME_CONVERSATION_TITLE);
+      setConversationId(getConversationId());
       assistantTelemetry?.reportAssistantInvoked({
         invokedBy: 'shortcut',
-        conversationId: localStorageLastConversationId ?? WELCOME_CONVERSATION_TITLE,
+        conversationId: getConversationId(),
```

Comment:
Thanks for ensuring all code paths return a consistent id with the new `getConversationId()` function!

The expected conversation id is included when the assistant is launched from the:
💹 Alerts flyout / page
💹 Data quality results / page
💹 Cases page
💹 Timeline
...and other contexts.

💹 The expected conversation ID is included for both default (e.g. `Alert summary`) and `Custom` conversations